airbyte-integrations/connectors/source-gorgias-api/source_gorgias_api/gorgias_api.yaml

```diff
@@ -0,0 +1,36 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://airbyte.gorgias.com"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "Authorization"
+      api_token: "Basic {{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
It is missing the pagination: https://developers.gorgias.com/reference/pagination